.codeclimate.yml

```diff
@@ -9,7 +9,7 @@ engines:
     config:
       languages:
         javascript:
-          mass_threshold: 100
+          mass_threshold: 130
```

Comment:
action modals all look the same, but they can't not (quite yet) look the same, hoping the work imma doing today is gonna address that